### DISCUSSION TITLE
SAML metadata: Unusual expiration date

### DISCUSSION THREAD

**Discussion by gabriel-hoff-vectorsolutions** [zitadel/zitadel#7500] [Q&A]
When creating a SAML IDP for an organization, the metadata XML file can be obtained via `https://{ZITADEL_CLOUD_DOMAIN}/idps/{IDP_ID}/saml/metadata`. The downloaded file contains an x509 certificate. When validating the certificate with [openssl](https://www.openssl.org/docs/man1.1.1/man1/openssl.html) command, the following expiration is returned:

```
Validity
    Not Before: Jan  1 00:00:00 1 GMT
    Not After : Jan  1 00:00:00 1 GMT
```

Does this mean the certificate is not valid? Or does it mean it never expires?

### Steps to reproduce:

1. Download the SAML metadata of the IDP
2. Find and copy the certificate by copying the contents inside the `<X509Certificate>` tag
3. Create a file named `certificate.crt` in a directory of your preference with the following contents:
    ```
    -----BEGIN CERTIFICATE-----
    {X509_CERTIFICATE}
    -----END CERTIFICATE-----
    ```
    Replace `{X509_CERTIFICATE}` with the certificate you copied from the SAML metadata file
4. Open a terminal and run the following command to see the certificate details:
    ```bash
    openssl x509 -in certificate.crt -text -noout
    ```

**Comment by hifabienne** [comment]
@livio-a @muhlemmer @stebenz Can you help here?

**Comment by stebenz** [comment]
@vs-gtadeu This means that the certificate doesn't expire, to be exact that this 2 dates are empty.
